build-release

```diff
@@ -5,12 +5,20 @@
 # ARCHITECTURE: architecture for the build
 # ISSUE_NUMBER: the number of the issue that triggered the build
 # DRY_RUN_PARAM: parameter that to use dry-run
+# ONLY_BUILD_TOOLCONF: if True, tells me to only build cmssw-tool-conf 
 
 HERE=$WORKSPACE/build
 BUILD_DIR=$HERE/$CMSSW_X_Y_Z-build
 mkdir -p $BUILD_DIR
 QUEUE=$CMSSW_QUEUE
 
+if [ "X$ONLY_BUILD_TOOLCONF" = Xtrue ]; then
+  POST_ACTION='TOOL_CONF_BUILDING'
+else
+  POST_ACTION='BUILDING'
+fi
+
+
 git clone git@github.com:cms-sw/cmsdist.git $BUILD_DIR/CMSDIST
```

Comment:
There was a similar issue for the upload. The upload is run in mesos, it ssh the machine that did the build and starts the upload. 
And I will add the tool-conf area to the cleanup.  The cleanup looks in the issue for which were the machines used for the build, now it can also look for the machines that did the tool-conf build and trigger a cleanup for them.